<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuakeSee</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tab_main">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_select_map">
        <attribute name="title">
         <string>Select Event</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MplCanvasMapWithToolbar" name="mpl_select_map" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="DateTimeEdit" name="datetime_start">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2023</year>
               <month>2</month>
               <day>6</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yy HH.mm.ss</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::LocalTime</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DateTimeEdit" name="datetime_end">
             <property name="dateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2023</year>
               <month>2</month>
               <day>6</day>
              </datetime>
             </property>
             <property name="date">
              <date>
               <year>2023</year>
               <month>2</month>
               <day>6</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>dd/MM/yy HH.mm.ss</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="timeSpec">
              <enum>Qt::LocalTime</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_rectangle">
             <property name="text">
              <string>Create Rectangle</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_showevents">
             <property name="text">
              <string>Show Events</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_selectevent">
             <property name="text">
              <string>Select An Event</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_map_reset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_select_plot">
        <attribute name="title">
         <string>Select Event (Plot)</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MplCanvasMapWithToolbar" name="mpl_select_map_plot" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btn_selectevent_plot">
             <property name="text">
              <string>Select An Event</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_stations">
        <attribute name="title">
         <string>Select Stations</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MplCanvasMapWithToolbar" name="mpl_select_map_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="btn_rectangle_2">
             <property name="text">
              <string>Create Rectangle</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_showstations">
             <property name="text">
              <string>Show Stations</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_selectstations">
             <property name="text">
              <string>Select Stations</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_getwaves">
             <property name="text">
              <string>Get Waveforms</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_showwaves">
             <property name="text">
              <string>Show Waveforms</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tab_code">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_console">
        <attribute name="title">
         <string>Python Console</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_reset">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="PyConsole" name="py_console" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_editor">
        <attribute name="title">
         <string>Code Editor</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CodeEditor" name="py_editor" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_run">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MplCanvasBaseWithToolbar" name="cv_distance" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyConsole</class>
   <extends>QWidget</extends>
   <header>widgets.pyconsole</header>
   <container>1</container>
   <slots>
    <slot>reset_kernel()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QWidget</extends>
   <header>widgets.codeeditor</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MplCanvasMapWithToolbar</class>
   <extends>QWidget</extends>
   <header>widgets.mplcanvas</header>
   <container>1</container>
   <slots>
    <slot>select_rectangle(bool)</slot>
    <slot>select_event(bool)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>widgets.datetimeedit</header>
  </customwidget>
  <customwidget>
   <class>MplCanvasBaseWithToolbar</class>
   <extends>QWidget</extends>
   <header>widgets.mplcanvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_reset</sender>
   <signal>clicked()</signal>
   <receiver>py_console</receiver>
   <slot>reset_kernel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_run</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_kernel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_rectangle</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpl_select_map</receiver>
   <slot>select_rectangle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_showevents</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_show_events_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_selectevent</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpl_select_map</receiver>
   <slot>select_event(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_rectangle_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpl_select_map_2</receiver>
   <slot>select_rectangle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_showstations</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_show_stations_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_selectstations</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_stations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_getwaves</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_get_waveforms_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_showwaves</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_waveforms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_selectevent_plot</sender>
   <signal>toggled(bool)</signal>
   <receiver>mpl_select_map_plot</receiver>
   <slot>select_event(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_map_reset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_map_reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>run_kernel()</slot>
  <slot>show_events()</slot>
  <slot>show_stations()</slot>
  <slot>select_stations()</slot>
  <slot>get_waveforms()</slot>
  <slot>show_waveforms()</slot>
  <slot>on_btn_get_waveforms_clicked()</slot>
  <slot>on_btn_show_stations_clicked()</slot>
  <slot>on_btn_map_reset()</slot>
  <slot>on_btn_show_events_clicked()</slot>
 </slots>
</ui>
